src/components/device-page/index.tsx

```diff
@@ -57,7 +57,7 @@ export class DevicePage extends Component<DevicePageProps & GlobalState, {}> {
             <h3>{device.friendly_name}</h3>
             <ul className="nav nav-tabs">
                 <li className="nav-item">
```

Comment:
Expected order: About, Exposes, Bind, Reporting, State, Clusters

Reply:
that was the first suggestion, but later sjorge suggested this order

https://github.com/nurikk/z2m-frontend/issues/210#issuecomment-752182782

Reply:
Expected order: About, Exposes, Bind, Reporting, State, Clusters